uui/src/lib/Box/box.svelte

```diff
@@ -23,23 +23,23 @@
 {#if $$slots.default}
   <svelte:element
     this={element}
-    use:scopedStyles={scopedStyle}
+    use:scopedStyles={{ ssx }}
     use:eventRedirection={eventComponents}
     bind:this={ref}
     {...$$restProps}
-    class="unstyled-box {$$restProps.class || ''}"
+    class="box unstyled-box {$$restProps.class || ''}"
```

Comment:
is box being used just for description? Didn't see an explicit box class anywhere.

Reply:
Just a reference class probably going to deprecate the unstyled-box on the release after next